marko/md_renderer.py

```diff
@@ -49,6 +49,7 @@ def render_list(self, element: "block.List") -> str:
             for child in element.children:
```

Comment:
```suggestion
            for num, child in enumerate(element.children, element.start):
```

A small optimization.